docs/vector/vector-intro.md

```diff
@@ -1,7 +1,7 @@
 # geotrellis.vector
 
->"Raster is faster but vector is correcter."
-— Somebody
+> “Yes raster is faster, but raster is vaster and vector just SEEMS more corrector.”
```

Comment:
This is actually in the geotrellis.raster intro! Makes it a funny dialog between geotrellis.vector and geotrellis.raster :)

https://github.com/geotrellis/geotrellis/blob/master/docs/raster/raster-intro.md#geotrellisraster